include/flyft/rpy_diffusive_flux.h

```diff
@@ -21,12 +22,13 @@ class RPYDiffusiveFlux : public Flux
         
         double getViscosity() const;
         void setViscosity(double viscosity);
-    
+        
     protected:
         void compute(std::shared_ptr<GrandPotential> grand, std::shared_ptr<State> state) override;
-        int determineBufferShape(std::shared_ptr<State> state, const std::string& type) override;
+        int determineBufferShape(std::shared_ptr<State> state, const std::string&) override;
```

Comment:
Revert this change. Although it is not strictly required by C++, I prefer to include the names of the variables in the header for readability.
```suggestion
        int determineBufferShape(std::shared_ptr<State> state, const std::string& type) override;
```